core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -30,39 +31,83 @@
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Tag;
 import io.micrometer.core.instrument.Timer;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
 
 /**
  * Provides utilities for accessing {@link MeterRegistry}.
  */
 public final class MoreMeters {
 
+    private static DistributionStatisticConfig distributionStatisticConfig;
+
+    public static void withConfig(DistributionStatisticConfig config) {
```

Comment:
- Missing Javadoc
- Rename to something like `setDistributionStatisticConfig()`?
- `requireNonNull(config, "config")`
- Add the getter as well: `public static DistributionStatisticConfig distributionStatisticConfig()`